manage/installing/installation_minimal_buildout.rst

```diff
@@ -0,0 +1,77 @@
+======================================
+Install Plone Using A Minimal Buildout
+======================================
+
+.. note ::
+
+   This example is adequate for a quick evaluation.
+
+   **Do not use it in production !**
+
+
+Create a directory called `Plone-5` and enter it:
+
+.. code-block:: shell
+
+   mkdir Plone-5
+   cd Plone-5
+
+Create a virtual python environment (`virtualenv <https://virtualenv.pypa.io/en/stable/>`_) and install `zc.buildout <https://pypi.python.org/pypi/zc.buildout>`_:
+
+.. code-block:: shell
+
+   virtualenv-2.7 zinstance
+   cd zinstance
+   bin/pip install zc.buildout
+
+Create a buildout.cfg file:
+
+.. code-block:: ini
+
+  echo """
+  [buildout]
+  extends =
+      http://dist.plone.org/release/5-latest/versions.cfg
+
+  parts =
+      instance
+
+  [instance]
+  recipe = plone.recipe.zope2instance
+  user = admin:admin
+  http-address = 8080
+  eggs =
+      Plone
+      Pillow
+
+  """ > buildout.cfg
+
+Run buildout:
+
+.. code-block:: shell
+
+   /bin/buildout
```

Comment:
I think a dot is missing here or we should remove the initial slash (this will be also coherent with the later usage of `bin/instance fg`).

Personal note: I would use for both the form `./bin/$COMMAND_NAME` because if dot (`.`) is not in the $PATH the command will fail.